nilearn/tests/test_extmath.py

```diff
@@ -50,3 +55,22 @@ def test_is_spd_with_symmetrical_matrix():
     matrix = np.array([[2, 1],
                        [1, 1]])
     assert is_spd(matrix, verbose=0)
+
+
+def test_fast_abs_percentile_no_index_error():
+    # Regression test
+    #####################
+
+    # check the offending low-level function
+    fast_abs_percentile(np.arange(4))
+
+
+def test_fast_abs_percentile_no_index_error_find_cuts():
+    # Regression test
+    #####################
+
+    # check that find_cuts functions are safe
+    iris = load_iris()
+    X, _ = iris.data, iris.target
+    X, _ = to_niimgs(X, [2, 2, 2])
+    nose.tools.assert_equal(len(find_xyz_cut_coords(mean_img(X))), 3)
```

Comment:
I don't think test_extmath.py is the right place to test find_xyz_cut_coords. The additional imports at the top of the file should be removed too.